File: sanjana/Day 8/discount.py
# Function to calculate final price after discount
def calculate_discount(original_price, discount_percentage):
    discount_amount = (original_price * discount_percentage) / 100
    final_price = original_price - discount_amount
    print("Discount Amount:", discount_amount)
    return final_price


# Main Program
original_price = float(input("Original Price: "))
discount_percentage = float(input("Discount Percentage: "))

final_price = calculate_discount(original_price, discount_percentage)

print("Final Price:", final_price)

File: nyasha/day8/Discount.java
import java.util.Scanner;

public class Discount {

    // Function to calculate final price after discount
    public static double calculate_discount(double originalPrice, double discountPercentage) {

        double discountAmount = (originalPrice * discountPercentage) / 100;
        double finalPrice = originalPrice - discountAmount;

        return finalPrice;
    }

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        // Taking input
        System.out.print("Original Price: ");
        double originalPrice = sc.nextDouble();

        System.out.print("Discount Percentage: ");
        double discountPercentage = sc.nextDouble();

        // Calling the function
        double finalPrice = calculate_discount(originalPrice, discountPercentage);

        // Calculating discount amount for display
        double discountAmount = (originalPrice * discountPercentage) / 100;

        // Displaying output
        System.out.println("Discount Amount: " + discountAmount);
        System.out.println("Final Price: " + finalPrice);

        sc.close();
    }
}